docs/faq.rst

```diff
@@ -161,6 +161,14 @@ If users are having trouble with PCA component estimation failing on a dataset,
 we recommend using RobustICA;
 and we invite users to send us feedback on its behavior and efficacy.
 
+Also note that ``--ica-method robustica`` can be paired with ``--n_robust_runs``.
```

Comment:
```suggestion
Also note that ``--ica_method robustica`` can be paired with ``--n_robust_runs``.
```

I think we only accept underscore-delimited arguments.

Reply:
Actually, to be more accurate, it looks like some of our parameters are underscore-delimited only and some are dash-delimited only, which isn't great.